### ISSUE TITLE
LFPy.Cell tstartms and tstopms arguments

### ISSUE THREAD

**Issue by espenhgn** [LFPy/LFPy#26]
Needs further investigation, but there appear to be issues with setting `tstartms` to values less than zero. This feature was implemented to easily ignore startup transients from simulation results in particular with active cell models

1. `cell.imem` is not equal to `seg.i_membrane_` values recorded during a simulation, but appear to be offset by `-dt`. 
2. Rename parameters `tstart` and `tstop`
3. Having these as LFPy.Cell parameters and attributes is a bit weird. Maybe they should be parameters for LFPy.Cell.simulate method
4. `cell.vmem` and `cell.somav` values is also wrong then for at values corresponding to t=0 (appear identical with cell.v_init)
5. Weirdly, extracellular potentials computed from `cell.imem` and during the main simulation loop with `RecExtElectrode` object supplied do not differ except for numerical precision

Solutions/remarks:

1. Remove this option altogether (as it could lead to questionable practice, never observing startup transients etc.)
2. Fix, but it may require reworking how simulations and recorders are initialized using NEURON. For example, the issue do not appear to be present in the following pure py-NEURON test script: 
```
        import numpy as np
        import neuron
        # sections
        soma = neuron.h.Section(name='soma')
        dend = neuron.h.Section(name='dend')
        
        # connect sections
        dend.connect(soma, 1, 0)
        
        # geometry
        soma.L = 30.
        soma.diam = 30.
        soma.nseg = 1
        dend.L = 500.
        dend.diam = 2.
        dend.nseg = 50
        
        # biophysical parameters
        for sec in [soma, dend]:
            sec.Ra = 100
            sec.cm = 1
            sec.insert('pas')
            for seg in sec:
                seg.pas.g = 0.0002
                seg.pas.e = -65.
                
        # stimulus
        syn = neuron.h.ExpSyn(0.5, sec=dend)
        syn.e = 0.
        syn.tau = 2.

        # generators
        ns = neuron.h.NetStim(0.5)
        ns.noise = 1.
        ns.start = 0.
        ns.number = 1000
        ns.interval = 10.
        nc = neuron.h.NetCon(ns, syn)
        nc.weight[0] = .01

        # integrator
        cvode = neuron.h.CVode()
        cvode.use_fast_imem(1)
        
        # record
        i_membrane_control = []     # record currents using Vector.record method
        i_membrane_fadvance = []    # record seg._i_membrane_ at each timestep
        for sec in [soma, dend]:
            for seg in sec:
                i = neuron.h.Vector()
                i.record(seg._ref_i_membrane_)
                i_membrane_control.append(i)
                i_membrane_fadvance.append([])
        
        # Simulation control
        neuron.h.dt = 2**-4          # simulation time resolution
        tstop = 500.        # simulation duration
        v_init = -65        # membrane voltage(s) at t = 0
        
        def initialize():
            neuron.h.finitialize(v_init)
            neuron.h.fcurrent()
            neuron.h.frecord_init()
            neuron.h.t = -100. # force simulations to start at some negative t

        def collect_i_membrane():        
            j = 0
            for sec in [soma, dend]:
                for seg in sec:
                    i_membrane_fadvance[j].append(seg.i_membrane_)
                    j += 1
                        
        def integrate():
            while neuron.h.t < tstop:
                collect_i_membrane()
                neuron.h.fadvance()
            collect_i_membrane() # otherwise shape mismatch 

        initialize()
        integrate()
        
        i_membrane_control = np.array(i_membrane_control)
        i_membrane_fadvance = np.array(i_membrane_fadvance)
        
        np.testing.assert_equal(i_membrane_control, i_membrane_fadvance)
```

**Comment by espenhgn** [comment]
The issue is now produced by a couple of tests in #27

**Comment by espenhgn** [comment]
Found out why this happens. If `tstartms < 0` and `dt` not in base 2, then a time shift may be introduced as NEURON and Python cannot exchange float numbers exactly as their internal representation differ.

**Comment by espenhgn** [comment]
Closing, fixed by PR #27
